### ISSUE TITLE
Problems with paramsFromQuery

### ISSUE THREAD

**Issue by kimdv** [p2/OAuth2#42]
In OAuth2Base i have some problems with 

```swift
public final class func paramsFromQuery(query: String) -> [String: String] {
		let parts = split(query.characters) { $0 == "&" }.map() { String($0) }
		var params = [String: String](minimumCapacity: parts.count)
		for part in parts {
			let subparts = split(part.characters) { $0 == "=" }.map() { String($0) }
			if 2 == subparts.count {
				params[subparts[0]] = subparts[1].wwwFormURLDecodedString
			}
		}
		
		return params
	}
```

It says following

p2.OAuth2/OAuth2/OAuth2Base.swift:203:15: 'split(_:maxSplit:allowEmptySlices:isSeparator:)' is unavailable: Use the split() method instead.

Any idea what is is?

Using the latest Xcode beta and iOS 9 beta on OS X 10.10.4

**Comment by kimdv** [comment]
```swift
public final class func paramsFromQuery(query: String) -> [String: String] {
		let parts = query.characters.split() { $0 == "&" }.map() { String($0) }
		var params = [String: String](minimumCapacity: parts.count)
		for part in parts {
			let subparts = part.characters.split() { $0 == "=" }.map() { String($0) }
			if 2 == subparts.count {
				params[subparts[0]] = subparts[1].wwwFormURLDecodedString
			}
		}
		
		return params
	}
```

I found this solution.

**Comment by p2** [comment]
You have to use the swift-2.0 branch with Xcode 7, current master is for Swift 1.2.

**Comment by p2** [comment]
Wait, you are. Weird, that should work, will take a closer look.

**Comment by p2** [comment]
Fixed in 7bc19b1, thanks for reporting!
